### PR TITLE
Enhancing bundling docs with command line instructions (AssetBundlerBatch)

### PR DESCRIPTION
**Pull request by AMZN-Olex**
Verified with Windows profile monolithic game build, 23.05 (/main branches of o3de, o3de-multiplayersample, o3de-multiplayersample-assets).

### CODE REVIEW

File: Documentation/PackedAssetBuilds.md [hunk 1]
```diff
@@ -120,6 +120,61 @@ Follow steps for "Create a bundle for game assets", "Create a bundle for engine
...
+    --assetListFile \path\to\o3de-multiplayersample\AssetBundling\AssetLists\game_pc.assetlist 
+    --outputBundlePath \path\to\o3de-multiplayersample\AssetBundling\Bundles\game_pc.pak 
+    --project-path \path\to\o3de-multiplayersample 
+    --allowOverwrites
+```
+
+The above commands will create two bundle files: `engine.pak` and `game_pc.pak` in `\path\to\o3de-multiplayersample\AssetBundling\Bundles` folder. These pak files will need to be copied along with the monolithic game launchers to create a stand-alone final game build.
+
+For 23.05 build of o3de-multiplayersample, the expected sizes of these pack files on Windows are:
+* `engine_pc.pak` - around 72 MB
```
Comment (lumberyard-employee-dm) [inline Documentation/PackedAssetBuilds.md:175]: Should this be `engine.pak`?
```suggestion
* `engine.pak` - around 72 MB
```

Comment (AMZN-Olex) [reply Documentation/PackedAssetBuilds.md:175]: The bundles are automatically suffixed with _<platform> even if not specified. So I left it as this to match the practical results.

File: Documentation/PackedAssetBuilds.md [hunk 2]
```diff
@@ -120,6 +120,61 @@ Follow steps for "Create a bundle for game assets", "Create a bundle for engine
...
+
+```shell
+build\windows\bin\profile\AssetBundlerBatch.exe bundles 
+    --assetListFile \path\to\o3de-multiplayersample\AssetBundling\AssetLists\game_pc.assetlist 
+    --outputBundlePath \path\to\o3de-multiplayersample\AssetBundling\Bundles\game_pc.pak 
+    --project-path \path\to\o3de-multiplayersample 
+    --allowOverwrites
+```
+
+The above commands will create two bundle files: `engine.pak` and `game_pc.pak` in `\path\to\o3de-multiplayersample\AssetBundling\Bundles` folder. These pak files will need to be copied along with the monolithic game launchers to create a stand-alone final game build.
```
Comment (tkothadev) [inline Documentation/PackedAssetBuilds.md:172]: I think if bundles are automatically suffixed with _ , then `engine.pak` should be updated to `engine_pc.pak`.

Comment (AMZN-Olex) [reply Documentation/PackedAssetBuilds.md]: Done

**Review by tkothadev — APPROVED** [review]

**Pull request merged**